.github/workflows/publish.js.yml

```diff
@@ -65,10 +80,72 @@ jobs:
         popd
         mv $PKG_PATH_TVOS/Build/Products/Debug-appletvos/$ZIP_PKG_NAME_TVOS ./
 
-    # release tasks
+    # building WDA packages for x86_64
```

Comment:
Runs sequential to push zip files with the semantics task at the end. We may be able to tune to run in multiple instances like .github/workflows/wda-package.yml for future work if needed

Reply:
can we make it to a matrix to avoid extra code duplication?

Reply:
let me try

Reply:
I tried to use local composit like Azure's template, but no succeeded yet.
`matrix` in strategy is not for this case since we should run all of them in one task to push materials in semantics-release for now.

Reply:
we can just create a depending job and run it after release is done
I don't see any particular reason to do everything in a single job

Reply:
It can when we do not push packages as part of semantics-release (currently semantics-release plugin does). So we should add a job to push files to a github release after the semantics-release task instead of building packages before the semantics-release job as current method. Then, maybe we can only use .github/workflows/wda-package.yml (but current release tasks do not so)